climada_petals/engine/supplychain.py

```diff
@@ -635,16 +749,144 @@ def calc_impacts(self,
                 axis=1,
             ).T.set_index(self.secs_shock.index)})
 
-        elif io_approach == "eeioa":
-            self.supchain_imp.update({io_approach : (
-                pd.DataFrame(
-                    self.secs_shock.dot(self.inverse[io_approach])
-                    * self.mriot.x.values.flatten()
-                )
-            )})
 
```

Comment:
I don't understand what happens if io_approach is set to "eeioa"? This is not treated in the if statement anymore?

Reply:
Yes, I decided to remove this option.
It was developed by Msc students working with Alessio and reading their thesis I found it very unconvincing. There's absolutely no scientific literature supporting it, and it is a "weird" mashup of Economic Footprint and risk analysis.

I decided that as it was not possible to properly document it and explain it in the tutorial, it should be removed until then.

Reply:
they haven't worked with me :-) .. fully aligned with your assessment @spjuhel ...